sql/core/benchmarks/CSVBenchmark-results.txt

```diff
@@ -2,66 +2,66 @@
 Benchmark to measure CSV read/write performance
 ================================================================================================
 
-OpenJDK 64-Bit Server VM 1.8.0_252-8u252-b09-1~18.04-b09 on Linux 4.15.0-1063-aws
-Intel(R) Xeon(R) CPU E5-2670 v2 @ 2.50GHz
+Java HotSpot(TM) 64-Bit Server VM 1.8.0_202-b08 on Mac OS X 10.15.7
```

Comment:
@HyukjinKwon I have older JVM ;-) BTW, can't you re-run the benchmark on the same EC2 instance?

Reply:
I don't think it matters. If we need to run it in the same machine with the same version, we should write down how to do that and force other people to do that. It will probably need a discussion .. this is non-trivial overhead, and I personally believe people won't like it very much ..

Reply:
> I don't think it matters.

You changed almost everything in the environment: CPU, memory, OS, JVM. Also you run the benchmark on your laptop, it seems. For instance, this impacts on `Stddev`, it became bigger because other processes (or some other activities in your laptop) influenced on the benchmark.

One more thing, if you look at existing benchmark results. Almost all of them were launched in the same environment. In my PRs and @dongjoon-hyun 's PRs, we pointed out the environment, see https://github.com/apache/spark/pull/30118, https://github.com/apache/spark/pull/28613.

Reply:
I disagree with pinning an env to test. That will necessarily bring overhead for the dev, and discourage other contributors to keep it updated. The purpose of benchmark codes is just to refer the numbers/ratios, and to make other developers easier to produce the numbers, etc.

Reply:
It would be great if we have a Docker image that keeps the exact same version like release container. In that way, there'd be virtually no overhead.

Reply:
IMHO, Docker image is not enough. Hardware matter too.

Reply:
@MaxGekk Could we provide a machine to run these benchmarks?

Reply:
See https://github.com/apache/spark/pull/28613, it is `r3.xlarge`.

Reply:
It seems this is not very convenient to use, especially for new contributors.